api/processors/group.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime
```

Comment:
When you import datetime this way, you are shadowing base datetime module, so I advise never do such import and instead of it write just `import datetime`. 

Very bad design solution in python standard library 🤦

Reply:
Fixed